src/service/file.js

```diff
@@ -13,18 +13,20 @@ export const uploadFile = async (base64String = '') =>
   });
 
 export const uploadPhoto = async (formData) => {
-  const result = await api.post('/upload/photo', formData, {
-    headers: {
-      'Content-Type': 'multipart/form-data'
-    },
-    testing: 'test'
-  });
+  try {
+    const result = await api.post('/upload/photo', formData, {
+      headers: {
+        'Content-Type': 'multipart/form-data'
+      },
+      testing: 'test'
+    });
 
-  if (result.data) {
-    return result.data;
+    if (result.data) {
+      return result.data;
+    }
+  } catch (e) {
+    return null;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a try-catch block is a good practice for error handling. However, simply returning `null` when an error occurs might not be the best approach. It would be more beneficial to log the error or throw it again after logging so that the caller function can handle it appropriately. This way, we can have more information about what went wrong during the execution.

```diff
  export const uploadPhoto = async (formData) => {
    try {
      const result = await api.post('/upload/photo', formData, {
        headers: {
          'Content-Type': 'multipart/form-data'
        },
        testing: 'test'
      });
  
      if (result.data) {
        return result.data;
      }
    } catch (e) {
-     return null;
+     console.error(e);
+     throw e;
    }
  };
```
This change will ensure that any errors encountered during the execution of the `uploadPhoto` function are logged and re-thrown, allowing for better error tracking and handling.


<!-- This is an auto-generated comment by OSS CodeRabbit -->